submissions/Levi123/html-css-popup/index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <nav class = "menu">
+        <ul class = "navigationList">
+            <li class = "elementList"><a href="#">Mail</a></li>
```

Comment:
```suggestion
            <li class="elementList"><a href="#">Mail</a></li>
```
You don't need spaces here

Reply:
Also please, read this [article](https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/) and use BEM for class names

Reply:
Thank you, i real article, but further study is required